src/components/modules/ui.ts

```diff
@@ -754,18 +752,46 @@ export default class UI extends Module<UINodes> {
     }
 
     /**
-     * Event can be fired on clicks at the Editor elements, for example, at the Inline Toolbar
-     * We need to skip such firings
+     * Usual clicks on some controls, for example, Block Tunes Toggler
      */
-    if (!focusedElement || !focusedElement.closest(`.${Block.CSS.content}`)) {
+    if (!focusedElement) {
+
+      /**
+       * If there is no selected range, close inline toolbar
+       *
+       * @todo Make this method more straightforward
+       */
+      if (!Selection.range) {
```

Comment:
as i can see, it already works on the release branch — when you collapse the sleection. the Inline Toolbar is closing. It is covered by `tryToShow` method. So please explain this change.

Reply:
`tryToShow` is called in this method on line 809. You've added this if statement with return in it, so if there is no `focusedElement` `tryToShow` is not called that's why we need additional check here